lib/user.rb

```diff
@@ -0,0 +1,15 @@
+class User
+  attr_reader :name, :lastname, :username, :email, :id
+
+  def initialize(name, lastname, username, email, password)
+    connection = PG.connect :dbname => "chitter_#{ENV['RACK_ENV']}"
+    insertion = connection.exec("INSERT INTO users (name, lastname, username, email, password) VALUES('#{name}', '#{lastname}', '#{username}', '#{email}', '#{password}') RETURNING id, name, lastname, username, email, password")
```

Comment:
It would be nicer to read this on several lines. You can look up ruby multiline strings as there are a number of solutions to this, and the one I give here may not be your favourite, but you could do:
```rb
query = %{
  INSERT INTO users (name, lastname, username, email, password)
  VALUES('#{name}', '#{lastname}', '#{username}', '#{email}', '#{password}')
  RETURNING id, name, lastname, username, email, password;
}
insertion = connection.exec(query)
```

Also this way you can easily log the query by adding a `puts query` between the query and insertion.